sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_tables_inventory_v1/query.py

```diff
@@ -39,16 +36,17 @@ def main():
     """Run query for each source project."""
     args = parser.parse_args()
 
-    partition = args.date.replace("-", "")
-    destination_table = f"{args.project}.{args.destination_dataset}.{args.destination_table}${partition}"
+    destination_table = (
+        f"{args.project}.{args.destination_dataset}.{args.destination_table}"
+    )
 
-    # remove old partition in case of re-run
+    # remove old table in case of re-run
     client = bigquery.Client(args.project)
     client.delete_table(destination_table, not_found_ok=True)
 
     for project in args.source_projects:
         client = bigquery.Client(project)
-        query = create_query(args.date, project)
+        query = create_query(project)
```

Comment:
I we replace the table on every run, does it still need to be `WRITE_APPEND`? Or would `WRITE_TRUNCATE` be correct?

Reply:
I think I need `WRITE_APPEND` still because I need to iterate through all data through all projects listed.

Reply:
Right